bus-mapping/src/evm/opcodes/begin_end_tx.rs

```diff
@@ -277,10 +277,8 @@ fn gen_end_tx_steps(state: &mut CircuitInputStateRef) -> Result<ExecStep, Error>
     if !found {
         return Err(Error::AccountNotFound(state.block.coinbase));
     }
-    let coinbase_account = coinbase_account.clone();
-    let coinbase_balance_prev = coinbase_account.balance;
+    let coinbase_exist = !coinbase_account.is_empty();
     let coinbase_transfer_value = effective_tip * (state.tx.gas() - exec_step.gas_left);
```

Comment:
Do we know when will the `coinbase_transfer_value` be zero? Is it when the signature or nonce was bad or tx.gas_limit was set to zero?

Reply:
@lispc said according to spec, the gas price can be zero, then value is zero. though it won't happen in real world

Reply:
I made a wrong assumption that the `coinbase_transfer_value` will never be zero so in the previous pr I didn't include this

Reply:
> @lispc said according to spec, the gas price can be zero, then value is zero.

Makes sense. Thanks for elaborating on it.